src/utility.jl

```diff
@@ -27,13 +27,40 @@ function update_opt_gap(m::PODNonlinearModel)
             m.best_rel_gap = 0.0
             return
         end
-        m.best_rel_gap = abs(m.best_obj - m.best_bound)/(m.tol+abs(m.best_obj))
+        if m.gapref == "ub"
```

Comment:
Again same comment - I feel this option is unnecessary @harshangrjn

Reply:
It all about benchmarking. Making it stop on an appropriate point seems to be a luxury option for many solvers.

Reply:
Okay, I am fine with this. What is the default?